components.html

```diff
@@ -4,9 +4,478 @@
+            </a>
+            <div class="navbar-links">
+                <ul>
+                    <li><h3>Components</h3></li>
+                    <li><a href="#installation-hai" style="font-weight:bolder;">Installation</a></li>
+                    <li><a href="#avatar-hai">Avatar</a></li>
+                    <li><a href="#alert-hai">Alert</a></li>
+                    <li><a href="#badge-hai">Badge</a></li>
+                    <li><a href="#button-hai">Button</a></li>
+                    <li><a href="#card-hai">Card</a></li>
+                    <li><a href="#image-hai">Image</a></li>
+                    <li><a href="#input-hai">Input</a></li>
+                </ul>
+            </div>
+        </navmobile>
+        <section class="installation" id="installation-hai">
+            <div class="main-nav">
+                Installation
+            </div>
+            Just copy and paste the link in your html file in between the head tag.
+               <div>
+                <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Feasywebdev.netlify.app%252Fcomponents.css%2522%253E"
+  style="width: 679px; height: 93px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </div>
+        </section>
+        <section class="avatar" id="avatar-hai">
+            <div class="main-nav">
+                Avatar
+            </div>
+                easyWEB provides very nice and easy to apply avatars of different size.
+                <div class="avatar-display">
+                    <img src="images/avatar.png" class="avatar-small" alt="">
+                    <img src="images/avatar.png" class="avatar-medium" alt="">
+                    <img src="images/avatar.png" class="avatar-large" alt="">
+                </div>
+                Small Avatar &nbsp; Medium avatar &nbsp;&nbsp; Large avatar
+               <div>
+                <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%252F%252Fsmall%2520avatar%250A%253Cimg%2520src%253D%2522images%252Favatar.png%2522%2520class%253D%2522avatar-small%2522%2520alt%253D%2522%2522%253E%250A%252F%252Fmedium%2520avatar%250A%253Cimg%2520src%253D%2522images%252Favatar.png%2522%2520class%253D%2522avatar-medium%2522%2520alt%253D%2522%2522%253E%250A%252F%252Flarge%2520avatar%250A%253Cimg%2520src%253D%2522images%252Favatar.png%2522%2520class%253D%2522avatar-large%2522%2520alt%253D%2522%2522%253E"
+  style="width: 527px; height: 186px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Inline css not preferred  , give width 100% to make it responsive and add CSS class for repeating  properties

Reply:
Thanks prathmesh for reviewing my code and thanks for pointing out my mistakes. This will help me in improving my html and css skills.